docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,131 @@
+  the SQL queries made during the lifespan of a request.
+- Templates panel: Presents you with the templates used when rendering the
+  response and the context used to render them.
+
+To install a third-party application like the toolbar, you need to install
+the package by running the command within an activated virtual environment.
+This is like our earlier step of :ref:`installing an official release of
+Django <installing-official-release>`.
+
+   .. console::
+
+        $ python -m pip install django-debug-toolbar
+
+.. admonition:: Managing application dependencies
+
+    Managing dependencies is an important part developing an application, but
+    it is outside of the scope of this tutorial. Please see the official Python
+    documentation regarding `Managing Application Dependencies
+    <https://packaging.python.org/en/latest/tutorials/managing-dependencies/>`_
+    for more information.
+
+Third-party packages that integrate with Django may need some setup.
+Sometimes you will need to add the package's Django app to your
+:setting:`INSTALLED_APPS` setting. `The Django Debug Toolbar has many
+installation steps
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+We're linking to the toolbar's documentation here because as a third-party
+package, it's maintained outside of the core development of Django. This
+allows the package maintainers to make releases outside of Django's release
+schedule
+
+Once installed, you should be able to see the DjDT "handle" on the right side
+of the browser window when you refresh the polls application.
+
+Getting help from others
+========================
+
+At some point you will run into a problem, for example the
+toolbar may not render. When this happens and you're unable to
+resolve the issue yourself, there are options available to you.
+
+#. If the problem is with a specific package, check if there's a
+   troubleshooting of FAQ in the package's documentation. For example the
+   Django Debug Toolbar has a `Tips section
+   <https://django-debug-toolbar.readthedocs.io/en/latest/tips.html>`_ that
+   outlines troubleshooting options.
+#. Consult the `Django Forum <https://forum.djangoproject.com/>`_.
+#. Join the `Django Discord server <https://discord.gg/xcRH6mN4fa>`_.
+#. Join the #Django IRC channel on `Libera.chat <https://libera.chat/>`_.
+
+Installing other third-party packages
+=====================================
+
+Another popular third-party package is `Django REST framework
+<https://www.django-rest-framework.org/>`_, also known as "DRF". This is a
+great package to use if you want to use your Django application in a headless
+design where Django only exposes an API while the frontend of your application
+is handled either in a mobile application or a JavaScript application.
+
+There are many more third-party packages than the Django Debug Toolbar and
+Django REST framework. You can find more third-party packages using the
```

Comment:
I think I would resist the urge to explicitly name another package here, even one such as DRF (which is ≈_frozen_ now, and who's to say how that evolves over the next cycle). 

Perhaps sufficient is something like: 

```suggestion
There are *many* more third-party packages than the Django Debug Toolbar, and
the other Jazzband projects. You can find more third-party packages using the
```

🤔

Reply:
Same here. I don't think we even need to mention "the other Jazzband"

```suggestion
There are many more third-party packages, which you can find using the
```

Reply:
Before I commit both of your suggestions, I would like to push back on this a little. Can you explain why we should be avoiding naming DRF explicitly here?

Reply:
Hey @tim-schilling, sure. 

* **General**: We've avoided linking to third-party packages ≈forever because they simple don't meet Django's level of stability, and we don't have capacity to keep maintaining references as the landscape changes. This policy has served Django well. Making an exception for DDT itself here is not done lightly: given its history it's reasonable to think it will be supported for as long as Django itself. (i.e. until the heat death of the universe ⚛️) I cite [JKL's 2010 article on favourite Django packages](https://jacobian.org/2010/nov/1/favorite-apps/) as early datapoint. I think we should be extremely cautious about extending that special licence. 
* **Specific**: DRF, whilst super popular, and very stable, and very reliable, is essentially in maintenance mode. It's not at all clear that it will still be the go-to option in its space even over the lifetime of Django's 5.x series, never mind beyond that. I think, given the general policy against linking to specific external packages, to link to it in a tutorial step at this point in its lifetime would be an error.

Reply:
Thank you @carltongibson for the explanation, I appreciate it.